management_api_app/db/events.py

```diff
@@ -10,14 +11,56 @@ async def connect_to_db(app: FastAPI) -> None:
     logging.debug(f"Connecting to {config.STATE_STORE_ENDPOINT}")
 
     try:
-        cosmos_client = CosmosClient(config.STATE_STORE_ENDPOINT, config.STATE_STORE_KEY)
+        cosmos_client = CosmosClient(config.STATE_STORE_ENDPOINT, config.STATE_STORE_KEY, connection_verify=False)
         app.state.cosmos_client = cosmos_client
         logging.debug("Connection established")
     except Exception as e:
         logging.debug(f"Connection to state store could not be established: {e}")
 
+# Bootstrapping is temporary while the API does not have a register spec api implemented.
+
+
+async def create_bundle_registry(database: DatabaseProxy):
+    """
+    Creates a bundle spec container if one does not exist and populate it with a canonical spec.
+    :param database: DatabaseProxy for STATE_STORE_DATABASE
+    :returns: None
+    """
+    bundle_spec = {
+        "url": "blah/blah:v1",
+        "id": str(uuid.uuid4()),
```

Comment:
Haven’t had a chance to run this yet. But… will this not add a new sample each time you start the API (as the UUID will make it a new record)

Reply:
This would also be solved by loading from disc as the sample would then have a predetermined UUID

Reply:
no before creation it checks if the container exists and if it does it skips. I did check it out and it works.